examples/model_selection/plot_learning_curve.py

```diff
@@ -2,16 +2,17 @@
 ========================
 Plotting Learning Curves
 ========================
-
-On the left side the learning curve of a naive Bayes classifier is shown for
+In the first row the learning curve of a naive Bayes classifier is shown for
```

Comment:
The rendering would be clearer with 3 rows and 2 columns as previously suggested.

Reply:
Indeed it is clearer using 3 rows and 2 columns (see 299d5d3)